common/app-starters-task-launch-request-common/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>app-starters-common</artifactId>
+		<groupId>org.springframework.cloud.stream.app</groupId>
+		<version>2.0.2.BUILD-SNAPSHOT</version>
```

Comment:
This is not your issue, but I think we are headed with this app for the Spring Cloud Stream 2.1 base line. Therefore the core for app starters should be `2.1.0` as well. 
I guess there is just no such a branch yet, but let's keep it in mind after moving there!
I hope the Maven magic for version bump will do the trick here as well 😄